src/app/pages/viewing/messages/messages.component.ts

```diff
@@ -34,9 +76,9 @@ export class MessagesComponent implements OnInit {
   deleteMessage() {
     // noinspection JSIgnoredPromiseFromCall
     this.viewService.msgIo.deleteMessage(this.messageToDelete).then(() => {
-      // TODO: Add a toast on successful deletion
+      this.toastr.success('Message deleted successfully', 'Poof!');
```

Comment:
I don't know if these are configurable, but I don't think having the toast in the top right corner is a good idea - especially since it obscures the navbar and isn't centred on mobile.
Maybe put them at the bottom of the screen like the cookie message?